contracts/renderers/CrowdfundNFTRenderer.sol

```diff
@@ -290,61 +288,24 @@ contract CrowdfundNFTRenderer is RendererBase {
 
     function getContribution(address owner) private view returns (string memory amount) {
         uint256 ethContributed;
-        if (getCrowdfundType() == CrowdfundType.NFT) {
-            (ethContributed, , , ) = Crowdfund(address(this)).getContributorInfo(owner);
-        } else {
-            uint96 votingPower = ReraiseETHCrowdfund(address(this)).pendingVotingPower(owner);
-
-            ethContributed = ReraiseETHCrowdfund(address(this)).convertVotingPowerToContribution(
-                votingPower
-            );
-        }
+        (ethContributed, , , ) = Crowdfund(address(this)).getContributorInfo(owner);
 
         return LibRenderer.formatAsDecimalString(ethContributed, 18, 4);
     }
 
     function getCrowdfundStatus() private view returns (CrowdfundStatus) {
-        if (getCrowdfundType() == CrowdfundType.NFT) {
-            Crowdfund.CrowdfundLifecycle lifecycle = Crowdfund(payable(address(this)))
-                .getCrowdfundLifecycle();
-
-            if (lifecycle == Crowdfund.CrowdfundLifecycle.Won) {
-                return CrowdfundStatus.WON;
-            } else if (
-                lifecycle == Crowdfund.CrowdfundLifecycle.Lost ||
-                lifecycle == Crowdfund.CrowdfundLifecycle.Expired
-            ) {
-                return CrowdfundStatus.LOST;
-            } else {
-                return CrowdfundStatus.LIVE;
-            }
-        } else {
-            ETHCrowdfundBase.CrowdfundLifecycle lifecycle = ETHCrowdfundBase(payable(address(this)))
-                .getCrowdfundLifecycle();
-
-            if (
-                lifecycle == ETHCrowdfundBase.CrowdfundLifecycle.Won ||
-                lifecycle == ETHCrowdfundBase.CrowdfundLifecycle.Finalized
-            ) {
-                return CrowdfundStatus.WON;
-            } else if (lifecycle == ETHCrowdfundBase.CrowdfundLifecycle.Lost) {
-                return CrowdfundStatus.LOST;
-            } else {
-                return CrowdfundStatus.LIVE;
-            }
-        }
-    }
-
-    function getCrowdfundType() private view returns (CrowdfundType) {
-        // If this function does not exist, then it is an NFT crowdfund.
-        (bool success, bytes memory res) = address(this).staticcall(
-            abi.encodeWithSignature("exchangeRateBps()")
-        );
-
-        if (success && res.length == 32) {
-            return CrowdfundType.ETH;
+        Crowdfund.CrowdfundLifecycle lifecycle = Crowdfund(payable(address(this)))
+            .getCrowdfundLifecycle();
+
+        if (lifecycle == Crowdfund.CrowdfundLifecycle.Won) {
+            return CrowdfundStatus.WON;
+        } else if (
+            lifecycle == Crowdfund.CrowdfundLifecycle.Lost ||
+            lifecycle == Crowdfund.CrowdfundLifecycle.Expired
+        ) {
+            return CrowdfundStatus.LOST;
         } else {
-            return CrowdfundType.NFT;
```

Comment:
Does this have to do with removing it?

Reply:
Yes, it's there because originally `ReraiseETHCrowdfund` minted crowdfund NFTs during the crowdfund before it was finalized.

Reply:
Without `ReraiseETHCrowdfund` it's no longer needed